src/IndexRegistry.php

```diff
@@ -29,7 +31,7 @@
  *
  * Provides an interface similar to Cake\ORM\TableRegistry.
  */
-class IndexRegistry
+class IndexRegistry implements LocatorInterface
```

Comment:
Made this registry compliant with our LocatorInterface which therefore removed all the static methods from this class.